<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScatterPointToolClass</class>
 <widget class="QMainWindow" name="ScatterPointToolClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1591</width>
    <height>1079</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ScatterPointTool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="2,2,1">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="label_lon">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>430</y>
            <width>211</width>
            <height>71</height>
           </rect>
          </property>
          <property name="text">
           <string>经度</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_lat">
          <property name="geometry">
           <rect>
            <x>230</x>
            <y>430</y>
            <width>241</width>
            <height>71</height>
           </rect>
          </property>
          <property name="text">
           <string>纬度</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_info">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>461</width>
            <height>141</height>
           </rect>
          </property>
          <property name="text">
           <string>输出信息</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_lon">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>280</y>
            <width>113</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_lat">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>280</y>
            <width>113</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_time">
          <property name="geometry">
           <rect>
            <x>340</x>
            <y>280</y>
            <width>113</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>250</y>
            <width>108</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>经度</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>250</y>
            <width>108</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>纬度</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>340</x>
            <y>250</y>
            <width>108</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>时间</string>
          </property>
         </widget>
         <widget class="QComboBox" name="comboBox">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>190</y>
            <width>451</width>
            <height>41</height>
           </rect>
          </property>
          <item>
           <property name="text">
            <string>WithLabel</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>WithoutLabel</string>
           </property>
          </item>
         </widget>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>330</y>
            <width>108</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>起始时间</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_8">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>330</y>
            <width>108</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>结束时间</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QPushButton" name="pBtn_start">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>260</y>
            <width>191</width>
            <height>91</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
}
QPushButton:hover{
	color:rgb(19,153,255);
	background-color:rgb(85,255,255);
}</string>
          </property>
          <property name="text">
           <string>开始</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pBtn_stop">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>260</y>
            <width>191</width>
            <height>91</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
}
QPushButton:hover{
	color:rgb(19,153,255);
	background-color:rgb(85,255,255);
}</string>
          </property>
          <property name="text">
           <string>结束</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pBtn_choosefile">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>60</y>
            <width>421</width>
            <height>131</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
}
QPushButton:hover{
	color:rgb(19,153,255);
	background-color:rgb(85,255,255);
}</string>
          </property>
          <property name="text">
           <string>选择文件</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>GPS度分格式转换成度格式</string>
         </property>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>291</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>输入GNGGA经纬度数据</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_gngga_lon">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>70</y>
            <width>181</width>
            <height>41</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_gngga_lat">
          <property name="geometry">
           <rect>
            <x>310</x>
            <y>70</y>
            <width>181</width>
            <height>41</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="pBtn_point">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>140</y>
            <width>241</width>
            <height>51</height>
           </rect>
          </property>
          <property name="text">
           <string>转换并撒点</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>61</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>经度</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>250</x>
            <y>70</y>
            <width>61</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>纬度</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWebEngineView" name="webEngineView">
         <property name="url">
          <url>
           <string>about:blank</string>
          </url>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ScatterPointTool.qrc"/>
 </resources>
 <connections/>
</ui>
